src/einsteinpy/metric/schwarzschild.py

```diff
@@ -110,7 +109,7 @@ def from_cartesian(cls, pos_vec, vel_vec, time, M):
     def christ_sym1_00(self, vec):
         num1 = (-2 * _G * self.M.value) + ((_c ** 2) * vec[1])
         num2 = _G * self.M.value
-        deno1 = _c ** 4
+        deno1 = _c ** 2
```

Comment:
Change in formula?

Reply:
> Change in formula?

The initial sources from which we got metric was faulty. I got the insight while developing Kerr Metric. That's why all scaling factors were needed to be introduced